contracts/Freeport.sol

```diff
@@ -25,7 +25,7 @@ contract Freeport is /*
 
     /** Initialize this contract and its dependencies.
      */
-    function initialize() public initializer {
-        __SimpleExchange_init();
+    function initialize(string memory baseURL) public initializer {
```

Comment:
The init functions should not change, instead there should be new ones. Then call the new function after upgrade.

Reply:
In fact, it’s simpler to just call it a setter rather than initialization.